### PR TITLE
Create iusearchbtw.json

### PR DESCRIPTION
**Pull request by iamnotmega**
reminder: X means yes

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [] You have completed your website. 
- [] The website is reachable.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

i have not created the website yet but ill do that once i get the domain

### CODE REVIEW

File: domains/iusearchbtw.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "subdomain": "iusearchbtw",
+  "owner": {
+    "email": "uksningaken@gmail.com"
+  },
```
Comment (Stef-00012) [inline domains/iusearchbtw.json:5]: Missing username field
```suggestion
  "owner": {
  	"username": "iamnotmega",
    "email": "uksningaken@gmail.com"
  },
```

File: domains/iusearchbtw.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "record": {
+    "CNAME": "iamnotmega.pages.dev"
```
Comment (Stef-00012) [inline domains/iusearchbtw.json:7]: I cannot reach your website
> iamnotmega.pages.dev’s server IP address could not be found.

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/iusearchbtw.json (hunk 2 above)
Comment (iamnotmega) [reply domains/iusearchbtw.json:8]: well yeah i havent made it yet

**Comment by iamnotmega** [comment]
there i updated the json to include username field now ill just make a really barebones website

**Comment by iamnotmega** [comment]
https://iamnotmega.github.io/iusearchbtw/

here is the link to the website @Stef-00012

**Pull request closed without merging**